src/tests.rs

```diff
@@ -47,6 +47,7 @@ void main() {
     assert!(compiler.compile_and_translate(&[source]).is_ok());
 
     let map = compiler.uniform_name_map();
+    println!("{:?}", map);
```

Comment:
Looks like a stray println again?

Reply:
All prints in tests.rs should indeed be removed, but I intended to keep prints in build.rs.